map_view/identify-layers/src/main/java/com/esri/samples/identify_layers/IdentifyLayersSample.java

```diff
@@ -73,14 +74,20 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
-      // create a map with an initial viewpoint
-      ArcGISMap map = new ArcGISMap(Basemap.createTopographic());
-      map.setInitialViewpoint(new Viewpoint(new Point(-10977012.785807, 4514257.550369, SpatialReference.create(3857)), 68015210));
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create a map with a basemap style
+      ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_TOPOGRAPHIC);
 
       // set the map to a map view
       mapView = new MapView();
       mapView.setMap(map);
 
+      // set a viewpoint on the map view
+      mapView.setViewpoint(new Viewpoint(new Point(-10977012.785807, 4514257.550369, SpatialReference.create(3857)), 68015210));
```

Comment:
Could use `SpatialReferences.getWebMercator()` here as well?

Reply:
Have removed the `Point` from this as per other changes.